### PR TITLE
[misc] Add Spanish translations to UIX_Fields

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

**Comment by veronikaslc** [comment]
Used Google translate, needs some proofread too

File: components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__solved.xml [hunk 1]
```diff
@@ -139,7 +139,7 @@
       <name>org.phenotips.patientSheet.field.solved</name>
     </property>
     <property>
-      <parameters>title=Case resolution
+      <parameters>title=$services.localization.render('phenotips.UIXField.solved')
```
Comment (sdumitriu) [inline components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__solved.xml]: Also update line 35.

File: components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__date_of_death.xml [hunk 2]
```diff
@@ -140,7 +140,7 @@
       <name>org.phenotips.patientSheet.field.date_of_death</name>
     </property>
     <property>
-      <parameters>title=Date of death
+      <parameters>title=$services.localization.render('phenotips.UIXField.dateOfDeath')
```
Comment (sdumitriu) [inline components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__date_of_death.xml]: Also update line 35.

File: components/patient-data/impl/src/main/resources/ApplicationResources_es.properties [hunk 3]
```diff
@@ -221,6 +222,7 @@ phenotips.UIXField.requiredMeasurements=Precauci\u00f3n, algunas medidas son req
 phenotips.UIXField.requiredMeasurements.warning=Se recomienda registrar el peso, altura y circunferencia de la cabeza para todos los pacientes.
 phenotips.UIXField.variants=Variaciones en el n\u00famero de copias
 phenotips.UIXField.geneVariants=Variantes gen\u00e9ticas
+phenotips.UIXField.solved=Resoluci\u00n de casos
```
Comment (sdumitriu) [inline components/patient-data/impl/src/main/resources/ApplicationResources_es.properties]: Ah, `\u00n` is the bad character that breaks the Spanish translation.

Comment (veronikaslc) [inline components/patient-data/impl/src/main/resources/ApplicationResources_es.properties]: My bad, will commit a fix today.

Comment (sdumitriu) [inline components/patient-data/impl/src/main/resources/ApplicationResources_es.properties]: Already on it.

**Pull request merged**